.github/actions/download.sh

```diff
@@ -19,3 +21,20 @@ else
         make -C ${download} download
     done
 fi
+
+echo ""
+
+# Download python wheel sources files
+build_packages="${NOARCH_PACKAGES} ${ARCH_PACKAGES}"
+
+if [ -z "${build_packages}" ]; then
+    echo "===> No wheels to download. <==="
+else
+    for package in ${build_packages}
+    do
```

Comment:
just for the sake of formatting ...
```suggestion
    for package in ${build_packages}; do
```